include/fastrtps/qos/LivelinessChangedStatus.h

```diff
@@ -0,0 +1,70 @@
+// Copyright 2019 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @file LivelinessChangedStatus.h
+*/
+
+#ifndef _LIVELINESS_CHANGED_STATUS_H_
+#define _LIVELINESS_CHANGED_STATUS_H_
+
+#include <fastrtps/rtps/common/InstanceHandle.h>
+
+namespace eprosima {
+namespace fastrtps {
+
+//! @brief A struct storing the liveliness changed status
+struct LivelinessChangedStatus
+{
+    //! @brief Constructor
+    LivelinessChangedStatus()
```

Comment:
You can remove it and use the implicit default constructor.
Also in C++11 you can initialize basic types in their definitions.

Reply:
:+1: